pygeos/constructive.py

```diff
@@ -486,6 +487,36 @@ def point_on_surface(geometry, **kwargs):
     return lib.point_on_surface(geometry, **kwargs)
 
 
+def polygonize(geometries, **kwargs):
+    """Creates polygons formed from the linework of a set of Geometries.
+
```

Comment:
The return value of `GeometryCollection` is a bit unexpected (though based on what GEOS provides), so it would be good to note this specifically.

```
Returns Polygons within a GeometryCollection.  
Individual Polygons can be obtained using get_parts, 
and MultiPolygons can be constructed from the output using
multipolygons(get_parts(polygonize(geometries)))


See also
--------
get_parts
```

(suggestion to use `multipolygons` and `get_parts` may be too specific, but I'd want an easy way to get outputs as Polygons or MultiPolygons)

Reply:
Yes, indeed, that's useful to highlight that the return is a collection and how to deal with it. See updated docstring.